addons/payment/tests/test_payment_transaction.py

```diff
@@ -11,7 +11,6 @@ class TestPaymentTransaction(PaymentCommon):
     def test_refunds_count(self):
         self.acquirer.support_refund = 'full_only'  # Should simply not be False
         tx = self._create_transaction('redirect', state='done')
-        tx._reconcile_after_done()  # Create the payment
```

Comment:
Why was this even needed if we can remove it just like that?

Reply:
Confirmed with EDM: probably due to the thought that refunds were made with the wizard, based on `account.payment` records.  But since the flow tested here does not go through the wizard, there is no need to have an `account.payment` for the test.